apps/src/applab/applab.js

```diff
@@ -680,7 +694,7 @@ Applab.init = function(config) {
     // should never be present on such levels, however some levels do
     // have levelHtml stored due to a previous bug. HTML set by levelbuilder
     // is stored in startHtml, not levelHtml.
-    if (config.level.hideDesignMode) {
+    if (Applab.reduxStore.getState().isDesignModeHidden) {
```

Comment:
Why?  As long as we're not mutating it, isn't it better to access the store than to duplicate state?

I can see the argument for hiding access to the store behind helper methods where we need to do this - and it will be inevitable in some places since not all of our code can ever be absorbed into React.

I see what you mean about this as a code smell though.  If I'm introducing too many of these in one PR, I may be refactoring poorly, moving the wrong state into redux.